python/viewer/image_viewer.py

```diff
@@ -304,17 +318,27 @@ def to_rgb_image(render_data):
     else:
         im_rgb = np.zeros((imsize_x, imsize_y, 3), dtype=np.uint8)
     if render_data.pc is not None:
+        cmap = cmap_doppler
+        if color_by_elevation:
+            cmap = cmap_elevation
         if render_data.image is not None:
             for pt in render_data.pc.point_cloud:
                 # use the image model to project ecef points to sar
                 x, y = render_data.image.image_model.global_to_image(pt.ecef)
```

Comment:
I am assuming that this `global_to_image` function already does the full 3D spherical-to-rectangular coordinate transformation?

Reply:
`global_to_image` does only the projection of the ECEf point on the image model. For the elevation I use `pt.xyz` which is the local cartesian position calculated from the full spherical coordinates: https://github.com/ZendarInc/ZendarSDK/blob/f445b18c6532e3c8dc17a426623e9708779ce344/python/viewer/radar_point_cloud.py#L60-L70